src/transaction/log_applier_sql_log.c

```diff
@@ -626,26 +620,38 @@ sl_open_next_file (FILE * old_fp)
 }
 
 /*
- * sl_remove_oldest_file() - remove oldest sql log file
- *   return: error code
+ * sl_remove_oldest_file_if_needed() - remove oldest sql log file
  *
  * Note:
  *   This function is related to the ha_sql_log_max_count system parameter.
  *   This system parameter can be set from 2 to 5 and only that number of sql log files are kept.
  */
-static int
-sl_remove_oldest_file (void)
+static void
+sl_remove_oldest_file_if_needed (void)
```

Comment:
NIT:

파일 삭제는 remove 보다는 delete가 더 적합해 보이네요.

```suggestion
sl_delete_oldest_file_if_needed (void)
```

Reply:
수정하였습니다.